src/modules/showLikes.js

```diff
@@ -0,0 +1,59 @@
+import {
+  getLikesData,
+  sendLikesData,
+} from './likesFetch.js';
+import likesData from './likesDataStore.js';
+
+export default class TheLikes {
+  likesStoredData = JSON.parse(localStorage.getItem('likesData'))
+
+  postLike = async (pid) => {
+    let newlike = {};
+    const item = (pid).toString();
+    newlike = {
+      item_id: item,
+    };
+    await sendLikesData(newlike);
+    // getLikesData().then((res) => {
+    //   likesData(res);
+    // });
+  }
+
+    getLikes = async () => {
+      const likedValues = await getLikesData()
+        .then((res) => {
+          res.sort((task1, task2) => task1.item_id - task2.item_id);
+          likesData(res);
+          return res;
+        });
+      return likedValues;
+    };
+
+    // likesUpdate = async (eId) => {
+    //   const likeS = this.likesStoredData;
+    //   getLikesData().then((res) => {
+    //     res.forEach((item) => {
+    //       likeS.forEach((like) => {
+    //         const ApiItem = res.find((item) => Number(item.item_id) === Number(eId));
+    //         const localItem = likeS.find((item) => Number(item.item_id) === Number(eId));
+    //         if (ApiItem.item_id === localItem.item_id) {
+    //           like.likes = item.likes;
+    //         }
+    //       });
+    //     });
+    //   });
+    // };
+
+    // sortLikes = () => {
+    //   const likes = this.likesStoredData;
+    //   likes.sort((task1, task2) => task1.item_id - task2.item_id);
+    // }
```

Comment:
:red_circle: [REQUIRED] Commented-out code is generally considered a bad practice in software development, and there are several reasons why it should be avoided:

1. It can make it harder for you and your team to read and understand the active code. Over time, as more commented-out code accumulates, it can become increasingly difficult to maintain and navigate the codebase.
2. Commented-out code can be misleading to other developers. When they see it, they may assume that it has some relevance to the current code or that it will be used in the future. This can lead to confusion and wasted time trying to understand the purpose of the commented-out code.
3. Version control systems (like Git) are designed to track changes to your code over time. When you commit commented-out code, it clutters your version history with unnecessary changes. This can make it more challenging to review the history of meaningful code changes and understand why specific changes were made.
4. ETC...

Reply:
Thank you again for the reminder.  
Commented file removed. 😄

Reply:
Can be improved further